CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ _ `_optional` subpackage for managing optional dependencies
 - `register_hooks` utility enabling user-defined augmentation of arbitrary callables
 - `transform` methods of `SearchSpace`, `SubspaceDiscrete` and `SubspaceContinuous`
   now take additional `allow_missing` and `allow_extra` keyword arguments
+- `SubspaceDiscrete.from_parameter`, `SubspaceContinuous.from_parameter` and
```

Comment:
I guess one of the docstrings is wrong resp. points to the wrong place, hence the docs are failing: 
<img width="863" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/aaaa4659-bdc0-4135-9b8a-6070179453f7">

Reply:
I think since it should be `core` instead of `base`?

Reply:
Yes, thx, fixed (haven't checked the CI yet, to be honest)